app/app.go

```diff
@@ -455,7 +455,7 @@ func New(
 
```

Comment:
Seems we are missing registering upgrade handler

Reply:
I didn't register it yet cos I figured there would be other changes - but yeah can do that, ezpz

Reply:
Going to resolve this as the handler should go in once 0.29 is finalized and in